rocket-chat-android-widgets/src/main/java/chat/rocket/android/widget/message/autocomplete/user/UserViewHolder.java

```diff
@@ -73,4 +83,35 @@ private String getImageUrl(String username, AbsoluteUrl absoluteUrl) {
       return null;
     }
   }
+
+  private Drawable getTextDrawable(Context context, String username) {
+    int round = (int) (4 * context.getResources().getDisplayMetrics().density);
+
+    return TextDrawable.builder()
+        .beginConfig()
+        .useFont(Typeface.SANS_SERIF)
+        .endConfig()
+        .buildRoundRect(getUsernameInitials(username), getUserAvatarBackgroundColor(username), round);
+  }
+
+  private String getUsernameInitials(String username) {
```

Comment:
Yes :(
I just put it as before (w/ some the correct modifications). This class belongs to another module, there is the possibility to add the `:app` dependency to this module to avoid that duplication, but I really don't know if this is the correct (in the terms of design, architecture...). So I just did like before.